MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -1,4 +1,5 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This line is an invalid add - most `.csproj` files do not have this file, and it should not be added here.